Lib/fontTools/ttLib/ttGlyphSet.py

```diff
@@ -75,7 +76,7 @@ def __getitem__(self, glyphName):
         return _TTGlyphCFF(self, glyphName)
 
 
-class _TTGlyph:
+class _TTGlyph(metaclass=ABCMeta):
```

Comment:
I tend to use the simpler `class _TTGlyph(ABC)` but this is basically the same, so it's ok

Reply:
Ah yes, that's better, thanks.